storage/src/vespa/storage/distributor/operations/idealstate/setbucketstateoperation.cpp

```diff
@@ -44,8 +45,8 @@ SetBucketStateOperation::shouldBeActive(uint16_t node) const
 
 void
 SetBucketStateOperation::activateNode(DistributorStripeMessageSender& sender) {
-    for (uint32_t i=0; i<_wantedActiveNodes.size(); ++i) {
-        enqueueSetBucketStateCommand(_wantedActiveNodes[i], true);
+    for (unsigned short _wantedActiveNode : _wantedActiveNodes) {
```

Comment:
Consider `uint16_t` and avoiding leading underscore for local variable

Reply:
hmm, I guess I have to double check when accepting CLions rewrites.
Fixed.